lib/internal/timers.js

```diff
@@ -482,7 +482,7 @@ function getTimerCallbacks(runNextTicks) {
 
     debug('timeout callback %d', msecs);
 
-    var diff, timer;
+    let diff, timer;
```

Comment:
Suggestions:
* The variable `diff` can be made a const, and definition can move to line `488`
* The variable `timer` can be made a const, and it's definition can move inside while loop to line `487`

@nikolaykrashnikov Can you test these suggestions?

Reply:
559:5  error  Unreachable code  no-unreachable

Reply:
`diff moved to line 488`
tests ok